docs/source/user_guide/using_datasets.rst

```diff
@@ -138,6 +138,17 @@ Datasets are homogeneous; they must contain samples of the same media type:
     dataset.add_sample(fo.Sample(filepath="/path/to/video.mp4"))
     # MediaTypeError: Sample media type 'video' does not match dataset media type 'image'
 
+Dataset version
+------------------
+
+The version of the `fiftyone` package with which a dataset is created is stored
```

Comment:
I might suggest "library" (also for other occurrences) for consistency, e.g. with index.rst

Reply:
I see package and library as interchangeable, except that I think we should use packgae when it is in specific reference to the `fiftyone` Python package or one of its attributes (like `version` here).

Reply:
I was considering this to refer to a more high-level "version of FiftyOne" than a low-level "`fiftyone.version`", but I guess either is fine with me.